spec/models/admin_user_tagging_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe AdminUserTagging, type: :model do
+  let(:admin_user) { create(:admin_user) }
+  let(:tag) { create(:person_tag) }
+
+  it 'validates non null fields' do
+    invalid = AdminUserTagging.new
+    expect(invalid).not_to be_valid
+    expect(invalid.errors.keys).to match_array(%i[
```

Comment:
Si son dos símbolos solamente, eq sirve.
```suggestion
    expect(invalid.errors.keys).to eq %i[admin_user tag]
```